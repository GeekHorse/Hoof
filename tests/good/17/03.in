new right one two done
left
word
value
word
value
right
word
value
word
value
save
