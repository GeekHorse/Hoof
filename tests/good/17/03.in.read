new 
ok 
ok 
ok   one 
ok   one two 
ok   one 
ok   one two 
ok 
ok   two 
ok   one two 
ok   two 
ok   one two 
ok 
