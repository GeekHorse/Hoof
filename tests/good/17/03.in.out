new right one two done
